From d32357c3e229fc6632037b1f4f3232b3fad41c2d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Verschelde?= <rverschelde@gmail.com>
Date: Fri, 10 May 2019 00:24:53 +0200
Subject: [PATCH] CMake: Properly support RelWithDebInfo build type

---
 cmake/BuildParameters.cmake | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/cmake/BuildParameters.cmake b/cmake/BuildParameters.cmake
index 262582814..4aeaa0981 100644
--- a/cmake/BuildParameters.cmake
+++ b/cmake/BuildParameters.cmake
@@ -133,7 +133,7 @@ endif()
 # Note without the CMAKE_BUILD_TYPE options the value is still defined to ""
 # Ensure that the value set by the User is correct to avoid some bad behavior later
 #-------------------------------------------------------------------------------
-if(NOT CMAKE_BUILD_TYPE MATCHES "Debug|Devel|Release|Prof")
+if(NOT CMAKE_BUILD_TYPE MATCHES "Debug|Devel|Release|RelWithDebInfo|Prof")
 	set(CMAKE_BUILD_TYPE Devel)
 	message(STATUS "BuildType set to ${CMAKE_BUILD_TYPE} by default")
 endif()
@@ -147,7 +147,7 @@ endif()
 option(CMAKE_BUILD_STRIP "Srip binaries to save a couple of MB (developer option)")
 
 if(NOT DEFINED CMAKE_BUILD_PO)
-    if(CMAKE_BUILD_TYPE STREQUAL "Release")
+    if(CMAKE_BUILD_TYPE MATCHES "Release|RelWithDebInfo")
         set(CMAKE_BUILD_PO TRUE)
         message(STATUS "Enable the building of po files by default in ${CMAKE_BUILD_TYPE} build !!!")
     else()
@@ -172,7 +172,7 @@ endif()
 include(TargetArch)
 target_architecture(PCSX2_TARGET_ARCHITECTURES)
 if(${PCSX2_TARGET_ARCHITECTURES} MATCHES "x86_64" OR ${PCSX2_TARGET_ARCHITECTURES} MATCHES "i386")
-	if(${PCSX2_TARGET_ARCHITECTURES} MATCHES "x86_64" AND (CMAKE_BUILD_TYPE MATCHES "Release" OR PACKAGE_MODE))
+	if(${PCSX2_TARGET_ARCHITECTURES} MATCHES "x86_64" AND (CMAKE_BUILD_TYPE MATCHES "Release|RelWithDebInfo" OR PACKAGE_MODE))
 		message(FATAL_ERROR "
         The code for ${PCSX2_TARGET_ARCHITECTURES} support is not ready yet.
         For now compile with -DCMAKE_TOOLCHAIN_FILE=cmake/linux-compiler-i386-multilib.cmake
@@ -365,7 +365,7 @@ if(CMAKE_BUILD_TYPE MATCHES "Debug")
     set(DEBUG_FLAG "${DBG} -DPCSX2_DEVBUILD -DPCSX2_DEBUG -D_DEBUG")
 elseif(CMAKE_BUILD_TYPE MATCHES "Devel")
     set(DEBUG_FLAG "${DBG} -DNDEBUG -DPCSX2_DEVBUILD -D_DEVEL")
-elseif(CMAKE_BUILD_TYPE MATCHES "Release")
+elseif(CMAKE_BUILD_TYPE MATCHES "Release|RelWithDebInfo")
     set(DEBUG_FLAG "-DNDEBUG")
 elseif(CMAKE_BUILD_TYPE MATCHES "Prof")
     # Keep frame pointer and debug information for profiler tool
-- 
2.21.0
